backend/lib/endpoints/posts.js

```diff
@@ -461,22 +564,22 @@ async function routes(app) {
     "/:postId/comments/:commentId/likes/:userId",
     { preValidation: [app.authenticate], schema: likeUnlikeCommentSchema },
     async (req) => {
-      const {
-        params: { commentId, postId },
-        userId,
-      } = req;
-      const updatedComment = await Comment.findOneAndUpdate(
-        { _id: commentId, likes: userId, postId },
-        { $inc: { likesCount: -1 }, $pull: { likes: userId } },
-        { new: true },
+      const { commentId, userId } = req.params;
+
```

Comment:
first this needs to check if `req.params.userId === req.userId` or you could remove someone else's like